guides/profitbricks.md

```diff
@@ -0,0 +1,476 @@
+			 */
+			DataCenter dc = api.dataCenterApi().createDataCenter(
+					DataCenter.Request.CreatePayload.create("JClouds", Location.DE_FKB)
+			);
+
+			/*  
+			 * DataCenterId: Defines the data center wherein the server is to be created.
+			 * AvailabilityZone: Selects the zone in which the server is going to be created (AUTO, ZONE_1, ZONE_2).
+			 * Cores: Number of cores to be assigned to the specified server. Required field.
+			 * InternetAccess: Set to TRUE to connect the server to the Internet via the specified LAN ID.
+			 * OsType: Sets the OS type of the server.
+			 * Ram: Number of RAM memory (in MiB) to be assigned to the server.
+			 */
+			String serverId = api.serverApi().createServer(Server.Request.creatingBuilder()
+					.dataCenterId(dc.id())
+					.name("jclouds-node")
+					.cores(1)
+					.ram(1024)
+					.build());
+
+			/*
+			 * The "waitUntilAvailable" server object method will wait until the server state is available before continuing
+			 */
+			waitUntilAvailable = Predicates2.retry(
+					new ProvisioningStatusPollingPredicate(api, ProvisioningStatusAware.SERVER, ProvisioningState.AVAILABLE),
+					2l * 60l, 2l, TimeUnit.SECONDS);
+
+			/*
+			 * Get list of all images.
+			 */
+			List<Image> images = api.imageApi().getAllImages();
+
+			/* dataCenterId: Defines the data center wherein the storage is to be created. If left empty, the storage will be created in a new data center
+			 * name: names the new volume.
+			 * Size: Storage size (in GiB). Required Field.
+			 */
+			String storageId = api.storageApi().createStorage(
+					Storage.Request.creatingBuilder()
+					.dataCenterId(dc.id())
+					.name("hdd-1")
+					.size(2f)
+					.build());
+
+			/*
+			 * ServerId: Id of the server to be updated.
+			 * ServerName: Renames target virtual server
+			 * Cores: Updates the amount of cores of the target virtual server
+			 * Ram: Updates the RAM memory (in MiB) of the target virtual 
+			 */
+			api.serverApi().updateServer(
+					Server.Request.updatingBuilder()
+					.id(serverId)
+					.name("apache-node")
+					.cores(2)
+					.ram(2 * 1024)
+					.build());
+
+			/*
+			 * id: identifier of a storage to be updated
+			 * name: changes the name of the storage
+			 */
+			api.storageApi().updateStorage(
+					Storage.Request.updatingBuilder()
+					.id(storageId)
+					.name("hdd-2")
+					.build());
+
+			/*
+			 * Disconnects storage from the server.
+			 */
+			api.storageApi().disconnectStorageFromServer(storageId, serverId);
+
+			/*
+			 * Fetches list of all DataCenters
+			 */
+			List<DataCenter> dataCenters = api.dataCenterApi().getAllDataCenters();
+
+			/*
+			 * Fetches list of all Volumes
+			 */
+			List<Storage> storages = api.storageApi().getAllStorages();
+
+			/*
+			 * Fetches list of all Servers
+			 */
+			List<Server> servers = api.serverApi().getAllServers();
+
+			api.dataCenterApi().deleteDataCenter(dc.id());
+		}
+	}
+	
+## pom.xml
```

Comment:
Why is the provider pom here? This is the pom for the profitbricks provider, but not the one users would use to build their apps using it.
I'd just remove this pom.